cips/cip-4.md

```diff
@@ -1,7 +1,7 @@
 ---
 cip: 4
 title: Standardize data expiry time for pruned nodes
-description: Standardize default data expiry time for pruned nodes to 30 days.
+description: Standardize default data expiry time for pruned nodes to 30 days + 1 hour worth of seconds (2595600 seconds).
```

Comment:
@renaynay I believe that some of these instances will revert back to just "30 days"